utils/utils.go

```diff
@@ -12,6 +11,8 @@ import (
 	"strings"
 	"syscall"
 
+	"golang.org/x/sys/unix"
```

Comment:
I have not include this package this package included automatically while saving file in vim-go. for more information about this package you can follow https://godoc.org/golang.org/x/sys/unix